pandoc.bzl

```diff
@@ -52,29 +52,48 @@ PANDOC_EXTENSIONS = {
 def _pandoc_impl(ctx):
     toolchain = ctx.toolchains["@bazel_pandoc//:pandoc_toolchain_type"]
     cli_args = []
+    all_outputs = [ctx.outputs.output]
+    all_data_inputs = []
+    all_data_inputs.extend(ctx.attr.data)
     cli_args.extend(ctx.attr.options)
     if ctx.attr.from_format:
         cli_args.extend(["--from", ctx.attr.from_format])
     if ctx.attr.to_format:
         cli_args.extend(["--to", ctx.attr.to_format])
+    if ctx.attr.css:
+        all_data_inputs.extend([ctx.attr.css])
+        cli_args.extend(["-c", ctx.file.css.path])
     cli_args.extend(["-o", ctx.outputs.output.path])
     cli_args.extend([ctx.file.src.path])
+    for target in all_data_inputs:
+        for df in target.files.to_list():
+            outfile = ctx.actions.declare_file(df.path)
```

Comment:
Should it be df.basename?

Reply:
Yes. thanks, it was fixed in a later commit that I had not pushed. Can you PTAL the whole change ?